src/core/manager.c

```diff
@@ -2610,23 +2610,23 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         uint32_t id;
 
                         if (safe_atou32(val, &id) < 0)
-                                log_debug("Failed to parse current job id value %s", val);
```

Comment:
hmm, not sure how I feel about this. The original intention was that errors like this are semi-expected during version updates, and not a problem nor actionable.